### PR TITLE
docs: instructions for using Jenkins Plugin via the UI

### PR DESCRIPTION
**Pull request by devcatalin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/articles/jenkins-ui.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Testkube Jenkins UI
+
+The Testkube Jenkins integration streamlines the installation of Testkube, enabling the execution of any [Testkube CLI](https://docs.testkube.io/cli/testkube) command within Jenkins Pipelines or Freestyle Projects.  
+
+If you're looking to use Pipelines and Groovy scripts, then look at examples from [Testkube Jenkins Pipelines](./jenkins.md).
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:5]:
```suggestion
If you are using Pipelines and Groovy scripts, then look at examples from [Testkube Jenkins Pipelines](./jenkins.md).
```

File: docs/docs/articles/jenkins-ui.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Testkube CLI Jenkins Plugin
+
+Install the Testkube CLI plugin by searching it in the "Available Plugins" section on Jenkins Plugins, or using the following url:
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:9]:
```suggestion
Install the Testkube CLI plugin by searching for it in the "Available Plugins" section on Jenkins Plugins, or using the following url:
```

File: docs/docs/articles/jenkins-ui.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[https://plugins.jenkins.io/testkube-cli](https://plugins.jenkins.io/testkube-cli)
+
+## Testkube Pro
+
+To use Jenkins CI/CD for [Testkube Pro](https://app.testkube.io/), you need to create an [API token](https://docs.testkube.io/testkube-pro/articles/organization-management/#api-tokens).
+
+
+### How to set up a Freestyle Project to run tests on Testkube Pro
+
+1. Create a new Freestyle Project
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:19]:
```suggestion
1. Create a new Freestyle Project.
```

File: docs/docs/articles/jenkins-ui.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+2. In General settings, configure the environment variables:
+   - TK_ORG
+   - TK_ENV
+   - TK_API_TOKEN
+  
+![jenkins environment variables configuration](../img/jenkins-environment.png)
+
+3. Click on "Add Build Step" and select "Testkube Setup"
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:27]:
```suggestion
3. Click on "Add Build Step" and select "Testkube Setup".
```

File: docs/docs/articles/jenkins-ui.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+![jenkins testkube setup build step](../img/jenkins-build-step.png)
+
+4. Specify a Testkube CLI version or leave it empty to use the latest version
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:30]:
```suggestion
4. Specify a Testkube CLI version or leave it empty to use the latest version.
```

File: docs/docs/articles/jenkins-ui.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+5. Add a new "Execute Shell" Build Step and run one or multiple Testkube CLI commands
```
Comment (jfermi) [inline docs/docs/articles/jenkins-ui.md:32]:
```suggestion
5. Add a new "Execute Shell" Build Step and run one or multiple Testkube CLI commands.
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by vsukhin — APPROVED** [review]

**Pull request merged**